src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/ContainerProcessorTest.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.blobrouter.tasks.processors;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.system.CapturedOutput;
+import org.springframework.boot.test.system.OutputCaptureExtension;
+import uk.gov.hmcts.reform.blobrouter.util.StorageClientsHelper;
+import uk.gov.hmcts.reform.blobrouter.util.StorageTestBase;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+@ExtendWith(OutputCaptureExtension.class)
+class ContainerProcessorTest extends StorageTestBase {
+
+    private static final String CONTAINER_WITH_BLOBS = "bulkscan";
+
+    private ContainerProcessor containerProcessor;
+
+    @Override
+    protected void beforeTest() {
+        containerProcessor = new ContainerProcessor();
+    }
+
+    @Test
+    @SuppressWarnings("java:S2925")
```

Comment:
what's this one about?

Reply:
it's about calling Thread.sleep() method in the tests. Added a comment now